INSTALL

```diff
@@ -112,40 +113,41 @@ Once the build files have been written, then compile with this command:
 
    cmake --build build
 
-Once the compilation is complete, install the game with this command:
+Once the compilation is complete, install the game with this command.
 
    cmake --install build
 
+   NOTE: If you did not change the default install prefix, you will need to elevate privileges with sudo.
+
 After compilation, the important results are:
 
   - The "build/freeciv-qt" and "build/freeciv-server" binaries.
   - The "build/data/" directory, which contains the graphics and scenarios.
 
-It's perfectly feasible to play Freeciv21 in this directory, without installing it.
-
 See the README file for more information.
 
-(However you cannot use Internationalization when playing Freeciv21 from the source directory; you must install Freeciv21 for that.)
-
 
 3. Debian Linux Notes:
 ======================
 
 Below are all the command line steps needed to start with a fresh install of Debian or its variants (e.g. Ubuntu, Linux Mint) to
 install Freeciv21.
 
-   sudo apt install build-dep-freeciv \
-     git \
+Start with ensuring your have a source repository (deb-src) turned on in apt sources and then run the following commands.
+
+   sudo apt update
+
+   sudo apt build-dep freeciv
+
+   sudo apt install git \
      cmake \
      ninja-build \
      python3 \
-     gettext \
      qt5-default \
      libkf5archive-dev \
-     libsqlite3-dev \
-     libsdl2-mixer-dev \
```

Comment:
`sdl2-mixer` is needed (but optional) for sound support.

Reply:
That is why I add it to the list of deps to install after the base `build-dep freeciv` step.

Reply:
Doesn't your patch remove it?

Reply:
Good catch. Fixed